src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -221,10 +221,11 @@ public boolean updatePatient(Patient patient) throws OHServiceException {
 	public boolean deletePatient(Patient patient) throws OHServiceException {
 		boolean isLoadProfilePhotoFromDB = LOAD_FROM_DB.equals(GeneralData.PATIENTPHOTOSTORAGE);
 		if (isLoadProfilePhotoFromDB) {
-			return  repository.updateDeleted(patient.getCode()) > 0;
+			repository.findById(patient.getCode()).get().setPatientProfilePhoto(null);
+			return repository.softDelete(patient.getCode()) > 0;
 		}
 		this.fileSystemPatientPhotoRepository.delete(GeneralData.PATIENTPHOTOSTORAGE, patient.getCode());		
-		return true;	
+		return repository.softDelete(patient.getCode()) > 0;
```

Comment:
the `PAT_ACTIVE` is not yet used because OP-780 is not yet merged (the `updateDeleted()` was already doing the soft deletion with `PAT_DELETED` field)

what about as follow (and rollback all the rest)?

```
if (isLoadProfilePhotoFromDB) {
	repository.findById(patient.getCode()).get().setPatientProfilePhoto(null);
} else {
	this.fileSystemPatientPhotoRepository.delete(GeneralData.PATIENTPHOTOSTORAGE, patient.getCode());		
}
return  repository.updateDeleted(patient.getCode()) > 0;
```